WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -2,6 +2,22 @@
 class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
+  def self.bad_keys_by_type_by_locale
+    @bad_keys_by_type_by_locale ||= begin
+      bad_keys_by_type_by_locale = {}
```

Comment:
Still a Ruby noob here. We've got 3 different things called `bad_keys_by_type_by_locale`: the function we're in, the class instance variable `@bad_keys_by_type_by_locale`, and the local variable `bad_keys_by_type_by_locale`. Kind of crazy. @larspetrus, @jonatanklosko, I'm curious if you think this is okay, or if you'd approach this differently.